### PR TITLE
Added current working directory option for launch_fluent API.

### PR DESCRIPTION
**Pull request by abhishekchitwar**
The following pull request aims to provide an option to specify current working directory so Fluent will get launched from the same.

Please check for the following enhancement request
 https://github.com/pyansys/pyfluent/issues/969

### CODE REVIEW

File: src/ansys/fluent/core/launcher/launcher.py [hunk 1]
```diff
@@ -536,6 +541,7 @@ def launch_fluent(
             if mode != LaunchModes.SOLVER_ICING:
                 env["APP_LAUNCHED_FROM_CLIENT"] = "1"  # disables flserver datamodel
             kwargs = _get_subprocess_kwargs_for_fluent(env)
+            kwargs.update(cwd=cwd)
```
Comment (mkundu1) [inline src/ansys/fluent/core/launcher/launcher.py:544]:
```suggestion
            if cwd:
                kwargs.update(cwd=cwd)
```

**Review by mkundu1 — APPROVED** [review]

File: src/ansys/fluent/core/launcher/launcher.py [hunk 2]
```diff
@@ -536,6 +541,8 @@ def launch_fluent(
             if mode != LaunchModes.SOLVER_ICING:
                 env["APP_LAUNCHED_FROM_CLIENT"] = "1"  # disables flserver datamodel
             kwargs = _get_subprocess_kwargs_for_fluent(env)
+            if cwd:
```
Comment (seanpearsonuk) [inline src/ansys/fluent/core/launcher/launcher.py:544]: @abhishekchitwar @mkundu1 is there a reason we can't simply add this to the json file?

Comment (abhishekchitwar) [reply src/ansys/fluent/core/launcher/launcher.py:544]: @seanpearsonuk, it seems JSON has all options related to Fluent specific options (as similar to Fluent command line / batch arguments). However, "cwd" is subprocess specific argument.

Comment (seanpearsonuk) [reply src/ansys/fluent/core/launcher/launcher.py:544]: @abhishekchitwar sorry, I didn't look properly before commenting, thanks.

**Review by seanpearsonuk — APPROVED** [review]

**Pull request merged**